src/mistral_common/tokens/tokenizers/tekken.py

```diff
@@ -250,7 +250,14 @@ def from_file(cls: Type["Tekkenizer"], path: Union[str, Path]) -> "Tekkenizer":
         untyped["special_tokens"] = special_tokens
 
         if mm := untyped.get("multimodal", None):
-            untyped["multimodal"] = MultimodalConfig(**mm)
+            # TODO(Patrick) - enable warning once fixed in vllm & transformers
+            # warnings.warn(
+            #     "multimodal is deprecated. Please rename your tekken config to use image instead.",
+            #     FutureWarning,
+            # )
```

Comment:
As we did for the special tokens, i do not think we should have a future warning. I believe that we should raise an error when the version will be > v11.